source/sgp_mode/CPU.h

```diff
@@ -45,19 +45,38 @@ class CPU {
   CPU(emp::Ptr<Organism> organism, emp::Ptr<SGPWorld> world,
       const sgpl::Program<Spec> &program)
       : program(program), state(organism, world) {
-    cpu.InitializeAnchors(program);
+    InitializeAnchors();
     state.self_completed.resize(world->GetTaskSet().NumTasks());
     state.shared_completed->resize(world->GetTaskSet().NumTasks());
   }
 
   void Reset() {
     cpu.Reset();
-    cpu.InitializeAnchors(program);
     state = CPUState(state.host, state.world);
+    InitializeAnchors();
     state.self_completed.resize(state.world->GetTaskSet().NumTasks());
     state.shared_completed->resize(state.world->GetTaskSet().NumTasks());
   }
-    
+
+  void InitializeAnchors() {
```

Comment:
Documentation for this?

Reply:
Good idea, I've added some documentation (and made this method private) in my fork now so that can be in the next PR.